rustls/src/client/handy.rs

```diff
@@ -163,6 +165,15 @@ impl client::ClientSessionStore for ClientSessionMemoryCache {
     }
 }
 
+impl Debug for ClientSessionMemoryCache {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
```

Comment:
Extremely low-grade nit: as above, I'd import `fmt` and then qualify `fmt::Debug`, `fmt::Formatter`, `fmt::Result`

Reply:
Fixed as well.